packages/bull-vault/test/integration-test/BullStrategyTestFork.t.sol

```diff
@@ -276,7 +278,7 @@ contract BullStrategyTestFork is Test {
         _deposit(crabToDeposit);
         vm.stopPrank();
 
-        uint256 bullToRedeem = crabToDeposit.sub(1e12);
+        uint256 bullToRedeem = crabToDeposit.sub(WETH_DECIMALS_DIFF);
```

Comment:
not supposed to be WETH_DECIMALS diff, should revert this change, just coincidentally 1e12